apps/web/src/containers/TradingContest/hooks/use-leaderboard-data.ts

```diff
@@ -47,59 +103,51 @@ export function useLeaderboardData() {
         },
       });
       const data: AccountResponse[] = await response.json();
-      const haveLeverage = data
+      const userData = data.find((c) => c.address === wallet?.selectedAddress);
+
+      const filteredHighRollerData = data
         .filter((a) => a.hasLeverage && a.irr)
-        .sort((a, b) => (b.irr || 0) - (a.irr || 0));
+        .filter(({ netDeposits }) => netDeposits > 100)
+        .sort((a, b) => (b.irr || 0) - (a.irr || 0))
+        .map((a, i) => formatTableData(a, i, true));
+
+      setHighRollerData(filteredHighRollerData);
+
+      const filteredFatCatData = data
+        .filter((a) => !a.hasLeverage && a.irr)
+        .filter(({ netDeposits }) => netDeposits > 100)
+        .sort((a, b) => (b.irr || 0) - (a.irr || 0))
+        .map((a, i) => formatTableData(a, i, false));
 
-      setHighRollerData(
-        haveLeverage.map((a, i) => ({
-          rank: (i + 1).toString().padStart(2, '0'),
-          username: {
-            text: truncateAddress(a.address),
-            dataSet: 'highRoller',
-            fullAddress: a.address,
-          },
-          address: a.address,
-          totalAPY: formatNumberAsPercent((a.irr || 0) * 100),
-          totalEarnings: `$${formatNumber(a.earnings)}`,
-          totalDeposits:
-            a.netDeposits < 0 ? `$0.0000` : `$${formatNumber(a.netDeposits)}`,
-          netWorth: `$${formatNumber(a.totalNetWorth)}`,
-        }))
-      );
-      setFatCatData(
-        data
-          .filter((a) => !a.hasLeverage && a.irr)
-          .sort((a, b) => (b.irr || 0) - (a.irr || 0))
-          .map((a, i) => ({
-            rank: (i + 1).toString().padStart(2, '0'),
-            username: {
-              text: truncateAddress(a.address),
-              dataSet: 'fatCat',
-              fullAddress: a.address,
-            },
-            address: a.address,
-            totalAPY: formatNumberAsPercent((a.irr || 0) * 100),
-            totalEarnings: `$${formatNumber(a.earnings)}`,
-            totalDeposits:
-              a.netDeposits < 0 ? `$0.0000` : `$${formatNumber(a.netDeposits)}`,
-            netWorth: `$${formatNumber(a.totalNetWorth)}`,
-          }))
-      );
+      setFatCatData(filteredFatCatData);
+
+      if (userData && userData.netDeposits < 100) {
```

Comment:
Instead of `netDeposits` this should be `totalNetWorth`. That applies for all of the places where you are filtering for `netDeposits`, sorry if that was unclear.